main/templates/base_home.html

```diff
@@ -5,7 +5,7 @@
     <title>High.Teach</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta charset="utf-8">
-    <link rel="stylesheet" type="text/css" href="{% static 'styles/loggedInPage.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'css/loggedInPage.css' %}">
```

Comment:
Same comment as this [PR](https://github.com/redhat-beyond/HighTeach/pull/84#discussion_r1195399933)